includes/Api/ApiTemplateDataHooks.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace MediaWiki\Extension\TemplateData\Api;
+
+/**
+ * This is a hook handler interface, see mediawiki/docs/Hooks.md.
+ */
+interface ApiTemplateDataHooks {
```

Comment:
where is this used?

Reply:
Here https://github.com/Wikia/unified-platform/pull/15869/files#diff-620fd9b5b921033514a8e2dc7e1bc32d87044cc74ee017f5fa8cbe6e052e1d36